platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -154,4 +131,74 @@ Properties loadProperties(String location) {
         return PropertiesLoader.loadProperties(location);
     }
 
+    private class CrlRefresh implements Runnable {
+
+        private Calendar start = Calendar.getInstance();
+
+        public void run() {
+            Properties encryptionProperties = loadProperties(encryptionPropertiesLocation);
+            CRL crl = crlCache.get();
+            if (crl instanceof X509CRL) {
+                X509CRL x509Crl = (X509CRL) crl;
+                if (1 == Calendar.getInstance()
```

Comment:
✏️ Clearer to do 
```java
if (Calendar.getInstance().getTime().after(x509Crl.getNextUpdate()))
```
Also, there's no guarantee that the `compareTo` call will return a `1` for after; it's just "some int greater than 0".